index.html

```diff
@@ -15,18 +15,30 @@
 </head>
 
 <body>
+  <header>
+    <h2>SpaceWalkers</h2>
+  <nav>
+    <ul>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">About</a></li>
```

Comment:
This is alot of unnecessary nesting. This will only make it harder for you to style and dig to in the Less. 

This should be 
```HTML
<div class="navbar">
    <h2>SpaceWalkers</h2>
    <nav>
      <a href="#">Home</a>
      <a href="#">About Us</a>
      <a href="#">Blog</a>
      <a href="#">Contact</a>
      <a href="#">Sign In</a>
    </nav>
  </div>

|or|


    
    <nav>
      <h2>SpaceWalkers</h2>
      <a href="#">Home</a>
      <a href="#">About Us</a>
      <a href="#">Blog</a>
      <a href="#">Contact</a>
      <a href="#">Sign In</a>
    </nav>


```

This would make things far easier for you in the styling, and tracking of properties.